### PR TITLE
add test runner image

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: tests-runner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+WORKDIR /tests-runner
+
+RUN apt-get update && apt-get install -y protobuf-compiler gcc-x86-64-linux-gnu curl bison npm python3-pip bash bsdmainutils
+
+# required for gvm
+SHELL ["/bin/bash", "-c"]
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o rustup-init.sh
+RUN chmod +x rustup-init.sh
+RUN ./rustup-init.sh -y -c rustfmt --default-toolchain nightly-2023-04-19
```
Comment (Razz4780) [inline tests-runner/Dockerfile:11]: Is `clippy` included by default?

File: .github/workflows/tests-runner-build.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+jobs:
+  docs-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
```
Comment (Razz4780) [inline .github/workflows/tests-runner-build.yaml:16]: Why do we need QEMU here?

**Review by Razz4780 — APPROVED** [review]

File: tests-runner/Dockerfile (hunk 1 above)
Comment (aviramha) [reply tests-runner/Dockerfile:11]: I don't think we need clippy for the e2e tests?

File: .github/workflows/tests-runner-build.yaml (hunk 2 above)
Comment (aviramha) [reply .github/workflows/tests-runner-build.yaml:16]: right

**Pull request merged**